Curvature/HTTPRequest/HTTPRequest.swift

```diff
@@ -32,6 +32,7 @@ public struct HTTPRequest {
 
     public var parameters: [String: String] = [:]
     public var data: [String: Any] = [:]
+    public var context: [String: Any] = [:]
```

Comment:
The `data` variable serves this purpose, but I'm thinking `context` is a better name. You can go ahead and delete the `data` variable.